join-metadata-and-clades.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+import argparse
+import re
+import sys
+import pandas as pd
+
+NEXTCLADE_JOIN_COLUMN_NAME = 'seqName'
+VALUE_MISSING_DATA = '?'
+
+column_map = {
+    "clade": "clade",
+    "outbreak": "outbreak",
+    "lineage": "lineage",
+    "coverage": "coverage",
+    "totalMissing": "missing_data",
+    "totalSubstitutions": "divergence",
+    "totalNonACGTNs": "nonACGTN",
+    "qc.missingData.status": "QC_missing_data",
+    "qc.mixedSites.status": "QC_mixed_sites",
+    "qc.privateMutations.status": "QC_rare_mutations",
+    "qc.frameShifts.status": "QC_frame_shifts",
+    "qc.stopCodons.status": "QC_stop_codons",
+    "frameShifts": "frame_shifts",
+    "isReverseComplement": "is_reverse_complement",
+#    "deletions": "deletions",
+#    "insertions": "insertions"
+#    "substitutions": "substitutions",
+#    "aaSubstitutions": "aaSubstitutions"
```

Comment:
_nitpick_

I'd prefer to completely remove these rather than leave them commented out.

Reply:
Agree, removed in 2ef8c4bdcf95f342fb8893f0f5ef5ed394dbb553